src/service/plugins/sftp.js

```diff
@@ -222,7 +222,7 @@ const SFTPPlugin = GObject.registerClass({
 
             // This is the actual call to mount the device
             const host = this.device.channel.host;
-            const uri = `sftp://${host}:${packet.body.port}/`;
+            const uri = `sftp://${host}:${packet.body.port}/sdcard/`;
```

Comment:
If I'm right, then this should be...

```suggestion
            const uri = `sftp://${host}:${packet.body.port}/${packet.body.path}/`;
```

Reply:
This looks correct if we're happy to access only the Android device's internal storage.

However, if I plug a USB thumb-drive into my phone, I get:

```json
"path": "/storage/emulated/0",
"multiPaths": [
  "/storage/emulated/0",
  "/mnt/media_rw/301B-13FB"
],
"pathNames": [
  "Internal storage",
  "Lexar USB drive"
]
```

We should probably mount each of these.

Reply:
@mavit 

Ugggh, that's really ugly. The worst part of it is, the `packet.body.path` is no longer even an _ancestor_ of both (all) of the `multiPaths` listed. I was hoping at least browsing `packet.body.path` and seeing both of the mounts under it would be possible.

Given the data you've provided, I agree — we should stop trying to browse a single "device filesystem root" at all, and just separately mount each of the `multiPaths` entries, however meany there are.

In theory we could have UI to select which one to mount, but that already feels more complicated than necessary. If the user has multiple filesystems exported on the device, and chooses to mount it, they should just get all of them.

But it's even more clear now that we should never try to browse **either** `sftp://${host}:${packet.body.port}/` **or** `sftp://${host}:${packet.body.port}/${packet.body.path}/`, since neither one represents the root of the remote filesystem. (There is, in fact, no path that **does**.)

Reply:
The really sucky thing is that, if I'm interpreting this right, there isn't going to be anything to indicate which mount point is which. I don't _think_ we can set volume labels on the remote mount points, and I'm not sure GVFS will pick them up from the KDEConnect side either.

I guess we can modify `_addSymlink` to create `${by_name_dir.get_path()}/${safe_device_name}` as a _directory_, instead of as the symbolic link itself, and then create one symlink for each mount point inside that directory. Then at least **we'll** have a mapping of the remote mount points to their names.